docs/src/hugo/content/getting-started/first-use.md

```diff
@@ -0,0 +1,162 @@
+
+<div class="alert alert-warning" role="alert">
+  Be aware that Nelson can only access repositories on the command line for which the provided Github access token used to <a href="/getting-started/install.html">login to Nelson</a> with, has access too.
+</div>
+
+To sync your repositories for the first time, run the following:
+
+```
+$ nelson repos sync
+```
+
+Depending upon the number of organizations and repositories you have access too, the time this operation takes to complete may very. For larger organizations with thousands of repositories this may take a several minutes, but in the common case it should be relatively quick. Once completed, you can list the repositories available in your system like so:
+
+```
+$ nelson repos list -o example
+```
+
+You should expect to see output that looks something like the following:
+
+```
+λ nelson repos list -o getnelson
+  REPOSITORY                OWNER      ACCESS  STATUS
+  api                       getnelson  admin   disabled
+  build-env                 getnelson  admin   disabled
+  cardinal                  getnelson  admin   disabled
+  cli                       getnelson  admin   disabled
+  containers                getnelson  admin   disabled
+  getnelson.github.io       getnelson  admin   disabled
+  [...]
+```
+
+The listed repos should (upon first use) all show as `disabled`. This indicates that Nelson deployments are *not* setup for that particular repository. In order to enable a repository for deployment, run the following command:
+
+```
+$ nelson repos enable \
+--organization getnelson \
+--repository hello-world
+```
+
+This command will instruct the Nelson server to install a Github webhook on the repository specified, which will send subsequent Github Deployment events to Nelson. From here, we can 
+
+## Triggering a Deployment
+
+In most organizations where Nelson is being deployed, [continuous deployment](https://en.wikipedia.org/wiki/Continuous_delivery#Relationship_to_continuous_deployment) is an explicit goal. That is to say, merging changes into the mainline branch should trigger a deployment into the targeted datacenter(s). Nelson is integrated with Github for this purpose; listening to [Github Deployments](https://developer.github.com/v3/guides/delivering-deployments/) as a trigger for action. Whilst you are free to trigger this however you like (Nelson doesn't care, provided the invokation sends the desired [deployable specification](/getting-started/deployables.html)), but to make this easy the Nelson project provides a supporting tool called [Slipway](https://github.com/getnelson/slipway). Checkout <a href="/downloads.html">the downloads page</a> to find the latest release of Slipway and Nelson CLI.
```

Comment:
Grammar: the sentence "Whilst you are free ..., but to make this easy ..." doesn't parse